etc/docker-scripts/bootstrapper.py

```diff
@@ -176,12 +176,10 @@ def put_file_to_object_storage(client, bucket_name, file_to_upload, object_name=
     get_file_from_object_storage(cos_client, input_params['cos-bucket'], input_params['cos-dependencies-archive'])
 
     print('Processing dependencies........')
-    if 'inputs' in input_params.keys():
+    if input_params['inputs']:
```

Comment:
this will fail if `inputs` is not on `input_params` with KeyError. If you indeed want to fail, i would say, send a message with exit(1) on the else?

Reply:
extra checks are good, but do we need this? when we add the inputs and outputs to the argument parser, this automatically adds the inputs and outputs keys to the `input_params` dict with a default value of `None`, which is considered falsy.

Reply:
passing in an explicity empty string of spaces '     ' would fail though..adding a check for that